internal/driver/driver_test.go

```diff
@@ -104,31 +104,29 @@ func TestParse(t *testing.T) {
 
 			testUI := &proftest.TestUI{T: t, AllowRx: "Generating report in|Ignoring local file|expression matched no samples|Interpreted .* as range, not regexp"}
 
-			f := baseFlags()
-			f.args = []string{tc.source}
+			f1 := baseFlags()
```

Comment:
Nit: could you just reassign the `f` variable below rather than renaming? It will make the diff a bit cleaner.

Reply:
Done.